### PR TITLE
Remove 1.8 references.

### PR DESCRIPTION
**Pull request by mterzo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,18 +20,10 @@ group :test do
     gem 'rspec', '~> 3.0'
     gem 'json'
   end
-
-  if RUBY_VERSION > '1.8'
-    # requires ruby 1.9+, on 1.8 we'll fall back to the old regex parsing
-    gem 'rspec-json_expectations', '~> 1.4'
```
Comment (rnelson0) [inline Gemfile]: This block should always be present, rather than removed.

File: Gemfile [hunk 2]
```diff
@@ -20,18 +20,10 @@ group :test do
...
-  end
 end
 
 group :development do
-  # For Changelog generation
-  if RUBY_VERSION > '1.9'
```
Comment (rnelson0) [inline Gemfile]: Does `> 1.9` cover 1.9.3 or only 2.0.0+?

File: Gemfile (hunk 1 above)
Comment (mterzo) [inline Gemfile]: You're correct. I missed that. Why did this not break during the unit tests?

File: Gemfile (hunk 2 above)
Comment (mterzo) [inline Gemfile]: I does cover 1.9.3. It was to prevent 1.8.  I'll double check from build logs.

File: Gemfile (hunk 1 above)
Comment (rnelson0) [inline Gemfile]: Because @binford2k was nice enough to politely skip that when the module doesn't load with a Rescue statement.

Comment (mterzo) [inline Gemfile]: Oh, maybe that should go away as well.

**Comment by rnelson0** [comment]
I'd like @binford2k to take a quick look and see if there's anywhere else the json_expectations would need adjustment.

**Pull request merged**